content/en/docs/reference/setup-tools/kubeadm/kubeadm-alpha.md

```diff
@@ -9,35 +9,6 @@ weight: 90
  from the community. Please try it out and give us feedback!
 {{< /caution >}}
 
-## kubeadm alpha kubeconfig user {#cmd-phase-kubeconfig}
-
-The `user` subcommand can be used for the creation of kubeconfig files for additional users.
-
-{{< tabs name="tab-kubeconfig" >}}
-{{< tab name="kubeconfig" include="generated/kubeadm_alpha_kubeconfig.md" />}}
-{{< tab name="user" include="generated/kubeadm_alpha_kubeconfig_user.md" />}}
-{{< /tabs >}}
-
-## kubeadm alpha kubelet config {#cmd-phase-kubelet}
-
-Use the following command to enable the DynamicKubeletConfiguration feature.
-
-{{< tabs name="tab-kubelet" >}}
-{{< tab name="kubelet" include="generated/kubeadm_alpha_kubelet.md" />}}
-{{< tab name="enable-dynamic" include="generated/kubeadm_alpha_kubelet_config_enable-dynamic.md" />}}
-{{< /tabs >}}
-
-## kubeadm alpha selfhosting pivot {#cmd-selfhosting}
-
-The subcommand `pivot` can be used to convert a static Pod-hosted control plane into a self-hosted one.
-
-[Documentation](/docs/setup/production-environment/tools/kubeadm/self-hosting/)
-
-{{< tabs name="selfhosting" >}}
-{{< tab name="selfhosting" include="generated/kubeadm_alpha_selfhosting.md" />}}
-{{< tab name="pivot" include="generated/kubeadm_alpha_selfhosting_pivot.md" />}}
```

Comment:
please delete the following generated files with this PR:
generated/kubeadm_alpha_selfhosting_pivot.md
generated/kubeadm_alpha_kubelet_config_enable-dynamic.md
generated/kubeadm_alpha_kubeconfig_user.md